new-improvements-to-redirects.rst

```diff
@@ -0,0 +1,75 @@
+.. post:: January 2, 2024
+   :tags: redirects, docs
+   :category: Changelog
+   :author: Santos
+   :location: CUE
+
+New improvements to redirects
+=============================
+
+As your documentation evolves, content is moved, and pages are renamed,
+leading to broken links for your users.
+Redirects allow you to point users to the new location of a page.
+
+We are excited to announce significant improvements to our redirects feature to make them more flexible and powerful.
+
+Key improvements
+----------------
+
+HTTP status code
+  You can now choose the `HTTP status code`_ for your redirects,
+  selecting between 301 (permanent redirect) and 302 (temporary redirect).
+  Previously, all redirects had the status code 302.
+
+.. _HTTP status code: https://developer.mozilla.org/en-US/docs/Web/HTTP/Redirections
+
+Explicit ordering
+  Redirects now have an explicit order,
+  allowing you to control the order in which they are processed.
+  Previously, redirects were ordered based on their last modification date.
```

Comment:
Did we ever make a change to Forced redirects with this work? I know we discussed something like:

* Always execute Exact redirects
* Only allow wildcard redirects on paid plans, since they are the more expensive ones to query

I think if possible getting rid of the Forced concept would be another really big win with this announcement, if we could do it.

Reply:
Edit: I see in the PR now that it's an open question... will address it here :)